run.js

```diff
@@ -82,25 +105,39 @@ function runWithOptions () {
   }
 }
 
+let csvIndex = argv.offset || 0
 let csvChunkIndex = 0
 function runOverCsv (chunks) {
   ids = chunks[csvChunkIndex]
 
   if (ids && ids.length > 0) {
```

Comment:
so chunks here would be the sorted groups from nyplSourceAwareChunks?

Reply:
If so, can you leave a comment describing why we're just checking the first id?

Reply:
Added additional inline comments (and slight refactoring) to try to clarify things